draft-ietf-quic-transport.md

```diff
@@ -1289,6 +1290,13 @@ preferred_address (0x0004):
 : The server's Preferred Address is used to effect a change in server address at
   the end of the handshake, as described in {{preferred-address}}.
 
+disable_migration (0x0009):
+
+: The server does not support migration of connections to a different IP
+  address. Clients MUST NOT send any packets, including probing packets, from an
+  IP address other than that used to perform the handshake.  This parameter
```

Comment:
This seems mildly ambiguous -- isn't there a scenario where the client doesn't necessarily have control over that IP? I'm not saying that that's the common case, but it could happen that the client doesn't just have a different port after a NAT rebinding, but that it actually shows up as coming from a different address entirely depending on how things are routed between NATs, especially after inactivity.

Reply:
I think most of my objection here relates to terminology. Perhaps: 
```
The server does not support connection migration ({{migration}}). 
Clients MUST NOT send any packets, including those intended for 
path validation ({{migrate-validate}}), from a local endpoint other
than that used to perform the handshake. ...
```

Reply:
My understanding is that one of the motives of having this option is to allow running QUIC servers behind a load balancer that distributes the load based on the 5-tuple.

Assuming that is true, it is necessary to minimize the chance of client rebinding to not only a different IP address but to a different port as well.